library/tasks/SmartSeq2PlateAggregation.wdl

```diff
@@ -4,7 +4,7 @@ task AggregateSmartSeq2Zarr {
     input {
         Array[Array[File]?] zarr_input
         String output_file_name
-        String docker = "quay.io/humancellatlas/secondary-analysis-ss2-plate-aggregation:0.0.1"
+        String docker = "quay.io/humancellatlas/secondary-analysis-ss2-plate-aggregation:0.0.1_np_test"
```

Comment:
We should create a new version of this docker without the test name. Maybe a minor version increase for this?